Tasks/Fedosenko/task_5/task_5.py

```diff
@@ -0,0 +1,49 @@
+
+def check_str(string_to_test):
+    counter_lower = 0
+    counter_upper = 0
+    if len(string_to_test) == 0:
+        print('Empty string')
+    else:
+
+        for char in string_to_test:
+            if char >= "a" and char <= "z":
+                counter_lower += 1
+            elif  char >= "A" and char <= "Z":
+                counter_upper += 1
```

Comment:
this logic is overcomplicated, there are specific function **isupper** and **islower**